src/overlays/actors/ovl_En_Ot/z_en_ot.c

```diff
@@ -41,128 +81,1101 @@ const ActorInit En_Ot_InitVars = {
+    this->actionFunc = func_80B5C3D8;
+}
+
+void func_80B5C3D8(EnOt* this, GlobalContext* globalCtx) {
+    static Color_RGB8 D_80B5E408 = { 255, 200, 80 };
+    static Color_RGB8 D_80B5E40C = { 255, 60, 200 };
+    s16 temp;
+    s32 pad;
+    Vec3f sp5C;
+    Vec3f sp50;
+    Vec3f sp44;
+    Vec3f sp38;
+
+    this->unk_3A0 += 0x2D8;
+    Matrix_RotateY(this->unk_3A0, 0);
+    SysMatrix_GetStateTranslationAndScaledZ(26.259998f, &sp5C);
+    SysMatrix_GetStateTranslationAndScaledZ(-26.259998f, &sp50);
+    this->unk_348.x = this->unk_394.x + sp5C.x;
+    this->unk_348.y = this->unk_394.y;
+    this->unk_348.z = this->unk_394.z + sp5C.z;
+
+    this->unk_360->unk_348.x = this->unk_394.x + sp50.x;
+    this->unk_360->unk_348.y = this->unk_394.y;
+    this->unk_360->unk_348.z = this->unk_394.z + sp50.z;
+
+    Math_Vec3f_Copy(&this->actor.world.pos, &this->unk_348);
+    Math_Vec3f_Copy(&this->unk_360->actor.world.pos, &this->unk_360->unk_348);
+    temp = Actor_YawBetweenActors(&this->actor, &this->unk_360->actor);
+    this->actor.shape.rot.y = temp;
+    this->unk_360->actor.shape.rot.y = BINANG_ROT180(temp);
+
+    if (this->unk_740 < 1.0f) {
+        Lib_LerpRGB(&D_80B5E408, &D_80B5E40C, this->unk_740, &this->unk_744);
+        Lib_LerpRGB(&D_80B5E408, &D_80B5E40C, this->unk_740, &this->unk_360->unk_744);
+        Math_StepToF(&this->unk_740, 1.0f, 0.05f);
+    }
+
+    if (func_801378B8(&this->skelAnime, 12.0f)) {
+        Matrix_RotateY(func_800DFCDC(ACTIVE_CAM), 0);
+        sp38.x = 1.0f;
+        sp38.y = 8.1f;
+        sp38.z = 0.0f;
+        SysMatrix_MultiplyVector3fByState(&sp38, &sp44);
+        Math_Vec3f_Sum(&this->unk_74C, &sp44, &sp44);
+        func_80B5DF58(this->unk_3A4, 1, &sp44, &this->actor.shape.rot, 10);
+        sp38.x = -1.0f;
+        sp38.y = 8.1f;
+        sp38.z = 0.0f;
+        SysMatrix_MultiplyVector3fByState(&sp38, &sp44);
+        Math_Vec3f_Sum(&this->unk_74C, &sp44, &sp44);
+        func_80B5DF58(this->unk_3A4, 2, &sp44, &this->actor.shape.rot, 10);
+    }
+}
+
+void func_80B5C634(EnOt* this, GlobalContext* globalCtx) {
+    this->actionFunc = func_80B5C64C;
+}
+
+void func_80B5C64C(EnOt* this, GlobalContext* globalCtx) {
+    if (gSaveContext.weekEventReg[26] & 8) {
+        Actor_MarkForDeath(&this->actor);
+    }
+}
+
+void func_80B5C684(EnOt* this, GlobalContext* globalCtx) {
+    this->actor.speedXZ = 0.0f;
+    func_8013E1C8(&this->skelAnime, sAnimations, 0, &this->animIdx);
+    this->actionFunc = func_80B5C6DC;
+}
+
+void func_80B5C6DC(EnOt* this, GlobalContext* globalCtx) {
+    Player* player = PLAYER;
+    s32 pad;
+    s16 sp3E;
+    Vec3f sp30;
 
-extern ColliderCylinderInit D_80B5E3A0;
-extern InitChainEntry D_80B5E3FC[];
+    sp3E = Actor_YawToPoint(&player->actor, &this->unk_394);
+    Matrix_RotateY(BINANG_ADD(sp3E, 0x4000), 0);
+    if (this->unk_33C == 2) {
+        SysMatrix_GetStateTranslationAndScaledZ(26.259998f, &sp30);
+    } else {
+        if (this->unk_73C == 0) {
```

Comment:
```suggestion
    } else if (this->unk_73C == 0) {
```

Reply:
This is non-equivalent due to the function call below.